### ISSUE TITLE
Delete tag from specific remote

### ISSUE THREAD

**Issue by drywolf** [ForkIssues/TrackerWin#1593]
We have GIT repositories that are forks of public github repositories, hence when we are working with Fork on those repositories we usually have 2 remotes for such a repository:

* origin --> points to our writable remote repo
* upstream --> points to the original github repo that we do not have write access to, only read access

If I create a new tag and want to push it to the remote, `Fork` asks me which remote I want to push the tag to, so I can push it just fine to our `origin` remote.
But when I want to delete such a tag that I created, I only have the checkbox option of `Delete tag from remote repositories`
`Fork` will then attempt to delete the tag from the `upstream` repo and fail with a GIT error (which is correct, since we will not have write permissions for that remote)

> remote: Permission to &lt;org&gt;/&lt;repo&gt;.git denied to &lt;username&gt;.

Currently there is no easy way to work around that.
Can we please have the option to select which remotes the tag should be deleted from 🙏 

Thanks

**Comment by drywolf** [comment]
PS: The same happens when creating a new tag, there is just a checkbox for `Push to all remotes`
I think it would be nice here also, to have a choice which remotes the tag should be pushed to.

**Comment by DanPristupov** [comment]
Yes, we should probably add the ability to select a particular remote.
